dependencies.yaml

```diff
@@ -76,8 +76,8 @@ ext-mbedtls:
     - MBEDTLS_FATAL_WARNINGS OFF  # disables setting warnings as errors  FIXME: workaround until upstream-fixes are included
 # everest-testing
 everest-utils:
-  git: https://github.com/EVerest/everest-utils.git
-  git_tag: v0.2.1
+  git: https://github.com/Qwello/everest-utils.git
```

Comment:
(we should not forget to switch this back to EVerest)

Reply:
Waiting to close the related PR, to get the right revision

Reply:
Updated